drink.py

```diff
@@ -60,15 +66,42 @@ def order_drink(drink, retry=True):
     if r.status_code == 403 and retry:
         refresh_token()
         return order_drink(drink, retry=False)
+
     if not r.ok:
-        print(str(r.status_code) + ": " + r.text, file=sys.stderr)
-        sys.exit(1)
-    print(r.text)
+        if r.status_code == 400:
+            availableDrinks = get_beverages()
+            smallestLD = 2000
+            correctName = ""
+            for d in availableDrinks:
+                ld = LD.distance(d["name"], drink)
+                if ld < smallestLD:
+                    smallestLD = ld
+                    correctName = d["name"]
+                elif ld == smallestLD:
+                    if drink in d["name"]:
+                        smallestLD = ld
+                        correctName = d["name"]
+
+            print("Did you mean ", end="")
+            print(correctName, end="")
+            print("? (y/n)")
+
+            answer = input()
+
+            if answer == "y":
+                order_drink(correctName)
+        else:
```

Comment:
consider inverting the corresponding if and returning to save one level of indentation

Reply:
done